py3status/modules/coin_price.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+
+""" Display CryptoCurrency using coinmarketcap.com
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module. A advise from the site:
+        "Please limit requests to no more than 10 per minute." (default 600)
+    coin_separator: display separator if more than one (default ', ')
+    coin_symbols: coin symbol that will display price (default 'btc,eth,xrp')
+    convert: currency unit want to display (default 'usd')
+    format: display format for this module (default '{format_coin}')
+    format_coin: display format for coins (default '{id}: {price}{symbol} ({percentage})')
+
+    format_coin placeholders:
+        {format_coin} format for coins
+
+    format_coin placeholders:
+        {id} coin's id (ex. bitcoin - btc, ethereum - eth)
+        {price} current prices
+        {symbol} currency symbols
+        {percentage} increment/decrement percentage during 24 hours
+"""
+
+import json
+
+try:
+    from urllib.request import urlopen
+except ImportError:
+    from urllib2 import urlopen
+
+
+class Py3status:
+    cache_timeout = 600
+    coin_separator = ', '
+    coin_symbols = "btc,eth,xrp"
+    convert = 'usd'
+    format = "{format_coin}"
+    format_coin = "{id}: {price}{symbol} ({percentage})"
+
+    def post_config_hook(self):
+        self.ticker = "https://api.coinmarketcap.com/v1/ticker/?convert={}".format(self.convert)
+
+        self.currency_map = {
+            'krw': '₩',
+            'aud': '$',
+            'cny': '¥',
+            'eur': '€',
+            'gbp': '£',
+            'usd': '$',
+            'jpy': '¥'
+        }
+
+        self.convert = self.convert.lower()
```

Comment:
i think user may give not only 'usd' or 'gbp' also but 'USD' or 'GBP' to convert argument